pants.toml

```diff
@@ -108,7 +108,7 @@ remote = "//:buildgrid_remote"
 
 [tailor]
 build_file_header = """\
-# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
+# Copyright 2023 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
I noticed this was out-of-date when I tried to use tailor to add new BUILD files. That could probably be a separate 1-character PR.